akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -56,7 +72,9 @@ object ActorTestKit {
    * The application.conf of your project is not used in this case.
    */
   def apply(name: String): ActorTestKit =
-    new ActorTestKit(name = TestKitUtils.scrubActorSystemName(name), config = ApplicationTestConfig, settings = None)
+    new ActorTestKit(
+      ActorSystem(ActorTestKitGuardian.testKitGuardian, TestKitUtils.scrubActorSystemName(name), ApplicationTestConfig),
```

Comment:
In this case the `ActorTestKitGuardian.testKitGuardian` isn't used, since it's always spawning another `internalTestKitGuardian`. It would be possible to get around that by passing in `testKitGuardian` in the constructor of ActorTestKit.

```
final class ActorTestKit private[akka] (val internalSystem: ActorSystem[_], internalTestKitGuardian: ActorRef[ActorTestKitGuardian.TestKitCommand], settings: Option[TestKitSettings]) {
```

then the two apply cases would look like:
```
  def apply(name: String): ActorTestKit = {
    val system = ActorSystem(ActorTestKitGuardian.testKitGuardian, TestKitUtils.scrubActorSystemName(name), ApplicationTestConfig)
    new ActorTestKit(
      system,
      system,
      settings = None)
  }

  def apply(system: ActorSystem[_]): ActorTestKit = {
    val testKitGuardian = system.systemActorOf(ActorTestKitGuardian.testKitGuardian, "test")
    new ActorTestKit(system, testKitGuardian, settings = None)
  }
```